easypost/constant.py

```diff
@@ -1,2 +1,7 @@
 SUPPORT_EMAIL = "support@easypost.com"
 TIMEOUT = 60
+
+FEDEX_ACCOUNT_TYPE = "FedexAccount"
+UPS_ACCOUNT_TYPE = "UpsAccount"
+
+_CUSTOM_WORKFLOW_CARRIER_ACCOUNT_TYPES = [FEDEX_ACCOUNT_TYPE, UPS_ACCOUNT_TYPE]
```

Comment:
NIT: no need to define the variables since they are only being used once, I would pass them in the list directly

Reply:
See: https://github.com/EasyPost/easypost-python/pull/245#discussion_r1029912613